### PR TITLE
Dropdown

### PR DESCRIPTION
**Pull request by jonotrujillo**

### CODE REVIEW

File: example/Dropdowns.jsx [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+      </Code>
+
+      <Subtitle>With Selected Value</Subtitle>
+      <Code>
+        <Dropdown label='Something went wrong' error options={options} value={2} />
+      </Code>
+
+      <SecondaryTitle>Warning</SecondaryTitle>
+      <Code>
+        <Dropdown label='Something is incorect' warning options={options} />
```
Comment (JoakimLofgren) [inline example/Dropdowns.jsx]: Typo: incorrect

File: components/Dropdown.jsx [hunk 2]
```diff
@@ -0,0 +1,139 @@
...
+              )
+            })
+          }
+        </select>
+      </div>
+    )
+  }
+}
+
+Dropdown.optionShape = PropTypes.shape({
```
Comment (JoakimLofgren) [inline components/Dropdown.jsx]: Each option had optional attributes for disabled and hidden as well right?

Comment (JoakimLofgren) [inline components/Dropdown.jsx]: Ah, you are just spreading all other ones.

Comment (jonotrujillo) [inline components/Dropdown.jsx]: Yup. What do you think about that? I'm not crazy about just spreading instead of explicitly setting them.

**Comment by xaviervia** [comment]
screen shot 2016-06-29 at 12 52 45

File: components/Dropdown.jsx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+import classNamesBind from 'classnames/bind'
+import defaultStyles from '@klarna/ui-css-components/src/components/dropdown.scss'
+import * as programmaticFocus from '../lib/features/programmaticFocus'
+import * as fieldStates from '../lib/features/fieldStates'
+import * as inlinedIcon from '../lib/features/inlinedIcon'
+import { position, size } from '../lib/features/stacking'
+import { handleKeyDown } from '../lib/features/keyboardEvents'
+
+export default class Dropdown extends Component {
+
```
Comment (joaomilho) [inline components/Dropdown.jsx]: doesn't lint complains about this space?

File: components/Dropdown.jsx [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+        <label className={classes.label}>{label}</label>
+        {
+          selectedOption &&
+            <div className={classes.currentOption}>{selectedOption.label}</div>
+        }
+        <select
+          className={classes.select}
+          disabled={disabled}
+          onBlur={onBlur}
+          onChange={onChange || function () {}}
```
Comment (joaomilho) [inline components/Dropdown.jsx]: why not having the `fn` as a default prop instead?

File: components/Dropdown.jsx (hunk 2 above)
Comment (joaomilho) [inline components/Dropdown.jsx]: The question I suppose is what do we gain w/ propTypes. If it's documentation, spreading doesn't help. If it's the console warn, fine. But I'm generally skeptic on propTypes ROI (the direction is good, but it doesn't deliver).

File: components/Dropdown.jsx (hunk 4 above)
Comment (xaviervia) [inline components/Dropdown.jsx]: Fixed in 403507d

**Pull request merged**